zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1831,6 +1873,85 @@ components:
           format: int64
           minimum: 1
 
+    FlowNodeInstanceSearchQueryRequest:
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryRequest"
+      type: object
+      properties:
+        filter:
+          allOf:
+            - $ref: "#/components/schemas/FlowNodeInstanceFilterRequest"
+    FlowNodeInstanceFilterRequest:
```

Comment:
I think we should add description to all these fields

Reply:
Thanks! `state` and `type` can also be enums. I propose to tackle this in another PR I need to revise the Operate related API definitions anyway. So there might be more changes to come.